### PR TITLE
Fix indentation in blog/2022/get-started-ambient/

### PR DESCRIPTION
**Pull request by windsonsea**
Give a consistent indentation with the suggested increment of 2 spaces on this blog page

- [x] Docs

### CODE REVIEW

File: content/en/blog/2022/get-started-ambient/index.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ Ambient mesh is [a new data plane mode for Istio introduced today](/blog/2022/in
 $ kind create cluster --config=- <<EOF
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-name: ambient
-nodes:
-- role: control-plane
-- role: worker
-- role: worker
+  name: ambient
```
Comment (kfaseela) [inline content/en/blog/2022/get-started-ambient/index.md:20]: This indentation is wrong and it does not work. name and nodes should be at the same level as apiVersion?

Comment (windsonsea) [reply content/en/blog/2022/get-started-ambient/index.md:20]: Not sure. I read the k8s doc yesterday:

https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

The YAML of a deployment looks like:

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: nginx-deployment
  labels:
    app: nginx
spec:
  replicas: 3
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:1.14.2
        ports:
        - containerPort: 80
```

Comment (zirain) [reply content/en/blog/2022/get-started-ambient/index.md:20]: this's a cluster, not deployment

Comment (zirain) [reply content/en/blog/2022/get-started-ambient/index.md:20]: sourcode is here: https://github.com/kubernetes-sigs/kind/blob/d53225750af998c6218b5c8c730b8b3ca0e2d472/pkg/apis/config/v1alpha4/types.go#L20

Comment (windsonsea) [reply content/en/blog/2022/get-started-ambient/index.md]: Understood. This block is changed back. 😸

File: content/en/blog/2022/get-started-ambient/index.md [hunk 2]
```diff
@@ -260,20 +259,20 @@ $ kubectl apply -f - <<EOF
...
-   - operation:
-       methods: ["GET"]
+  selector:
+    matchLabels:
+      app: productpage
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+      principals: ["cluster.local/ns/default/sa/sleep", "cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account"]
```
Comment (kfaseela) [inline content/en/blog/2022/get-started-ambient/index.md:272]: this one was aligned right previously. is that not needed? same with operation as well.

**Comment by windsonsea** [comment]
well, seems this PR is not necessary. It's time to close, thanks you all.

**Pull request closed without merging**